apps/site/redirects.json

```diff
@@ -332,6 +332,10 @@
       "source": "/:locale/download/source-code/current",
       "destination": "/:locale/download/current"
     },
+    {
```

Comment:
Since the file (`/pages/download/archive/index.mdx`) is under the pages directory, it maps to `/download/archive` (When we remove version control, it drops to 500 page because there is no version named `archive`). In fact, this file doesn’t really need to live under `pages` directory

- Store the dynamically generated markdown files in a different directory instead of `pages`
- Redirect `/download/archive` to `/download/archive/current`
- Use an existing version(eg: `v22.0.0.mdx`) as the filename instead of `index.mdx`

IDK wich one suits for us better, for a quick fix i added this redirect :shipit:

Reply:
IDK, instead of going for an overkill solution, I added back the release check for now